content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,364 @@
+
+Mutating admission policy is part of the cluster control-plane. You should write
+and deploy them with great caution. The following describes how to quickly
+experiment with Mutating admission policy.
+
+### Create a MutatingAdmissionPolicy
+
+The following is an example of a MutatingAdmissionPolicy. This policy mutates newly created Pods to have a sidecar container if it does not exist.
+
+```yaml
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: MutatingAdmissionPolicy
+metadata:
+  name: "sidecar-policy.example.com"
+spec:
+  paramKind:
+    kind: Sidecar
+    apiVersion: mutations.example.com/v1
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["apps"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE"]
+      resources:   ["pods"]
+  matchConditions:
+    - name: does-not-already-have-sidecar
+      expression: "!object.spec.initContainers.exists(ic, ic.name == \"mesh-proxy\")"
+  failurePolicy: Fail
+  reinvocationPolicy: IfNeeded
+  mutations:
+    - patchType: "ApplyConfiguration"
+      applyConfiguration:
+        expression: >
+          Object{
+            spec: Object.spec{
+              initContainers: [
+                Object.spec.initContainers{
+                  name: "mesh-proxy",
+                  image: "mesh/proxy:v1.0.0",
+                  args: ["proxy", "sidecar"],
+                  restartPolicy: "Always"
+                }
+              ]
+            }
+          }
+```
+
+`spec.mutations` consists of a list of patches. Patches may be either [ApplyConfiguration](#applyconfiguration) or [JSON Patch](#json-patch).
+
+To configure a mutating admission policy for use in a cluster, a binding is required. The following is an example of a MutatingAdmissionPolicyBinding:
+
+```yaml
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: MutatingAdmissionPolicyBinding
+metadata:
+  name: "sidecar-binding-test.example.com"
+spec:
+  policyName: "sidecar-policy.example.com"
+```
+
+Next, the policy can be tested by creating a pod:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  initContainers:
+  - name: myapp-initializer
+    image: example/initializer:v1.0.0
+  containers:
+  - name: myapp
+    image: example/myapp:v1.0.0
+```
+
+Since the API request to create it matches the MutatingAdmissionPolicy the pod will be mutated resulting in:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  ...
+  initContainers:
+  - name: mesh-proxy
+    image: mesh/proxy:v1.0.0
+    args: ["proxy", "sidecar"]
+    restartPolicy: Always
+  - name: myapp-initializer
+    image: example/initializer:v1.0.0
+  containers:
+  - name: myapp
+    image: example/myapp:v1.0.0
+```
+
+#### Parameter Resources
+
+Parameter resources allow a policy configuration to be separate from its
+definition. A policy can define paramKind, which outlines GVK of the parameter
+resource, and then a policy binding ties a policy by name (via policyName) to a
+particular parameter resource via paramRef.
```

Comment:
```suggestion
Parameter resources allow a policy configuration to be separate from its
definition. A policy can define `paramKind`, which outlines GVK of the parameter
resource, and then a policy binding ties a policy by name (via `policyName`) to a
particular parameter resource via `paramRef`.
```

At beta, we should explain what a GVK is. OK to add that clarification for alpha too.